src/core/workflow/nodes/http.js

```diff
@@ -90,6 +90,11 @@ class HttpSystemTaskNode extends SystemTaskNode {
           status: err.response.status,
           data: err.response.data,
         };
+      } else if(err.code === 'ECONNABORTED') {
```

Comment:
1. esse else não deveria estar condicionado ao conteúdo do valid_response_codes e não fixo? A regra como está descrita sempre segue com o result, mesmo que o usuário não tenha relacionado.
2. a validação de response_codes (incluindo o ECONNABORTED não deveria ser feita lá no requests, dentro do config do Axios)?

Reply:
1 - O retorno do AXIOS é um ECONNABORTED fixo para esse cenário, por isso adicionei esse "if" específico. O que ele faz é adicionar uma tratativa de erro que permite que o erro chegue no if seguinte de consulta dos valid_response_codes. Se ECONNABORTED não for relacionado, o processo continua caindo em erro, mas agora um de status inválido.
Segui um caminho de mudanças mínimas.

2 - Acho que podemos mudar onde é feito isso (tirando do _run do node para injetar a config nos métodos de request). No momento, essa é a mudança mínima que habilita o recurso.